frontend/packages/dev-console/integration-tests/testData/yamls/create-statefulsets.yaml

```diff
@@ -4,26 +4,26 @@ metadata:
   name: example
   namespace: default
 spec:
-  serviceName: nginx
+  serviceName: httpd
```

Comment:
Im wondering why we are not using the resource template that is part of the `yaml-templates.ts`